agency_api/api.py

```diff
@@ -299,3 +299,97 @@ def retrieve(self, request, pk):
         serializer = self.serializer_class(service_request, many=False)
 
         return Response(serializer.data)
+
+class HPJobApplicationViewSet(viewsets.ModelViewSet):
+    #permission_classes = [CustomModelPermissions]
```

Comment:
https://github.com/MarkyMan4/hiring-agency/pull/20/files

Reply:
> do we want this line be uncommented? Should access to these end points be restricted to certain roles? As is, this is a publicly accessible end point

Yes I think I commented it because I want to do some test on postman, now it can be uncommented